docs/content/authentication.md

```diff
@@ -5,47 +5,90 @@ title: "Authentication"
 
 # Authentication and Configuration
 
-Each system handles authentication differently. Authentication must be configured
-if resources are provided for a system.
+## Introducing Contexts
+Grizzly supports 'contexts' (much like `kubectl` contexts), allowing users to maintain
+configuration for multiple Grafana (and Mimir and Synthetic Monitoring) systems.
+
+By default, the `default` context is used. We will discuss how to use the default
+context first, then explain how to use multiple contexts thereafter.
+
+## Settings
+
+> **NOTE**: If you have used Grizzly previously with environment variables, you can
+  initialise the `default` context from your environment simply with `grr config init`.
+
+Settings can be configured via Grizzly itself, using `grr config set`. They are stored
+in an OS specific location.
 
 ## Grafana Itself
-This tool interacts with Grafana via its REST API. For this, you will need to
-establish authentication credentials. These are provided to `grr` via
-environment variables.
 
-| Name | Description | Required | Default |
-| --- | --- | --- | --- |
-| `GRAFANA_URL` | Fully qualified domain name of your Grafana instance. | true | - |
-| `GRAFANA_USER` | Basic auth username if applicable. | false | `api_key` |
-| `GRAFANA_TOKEN` | Basic auth password or API token. | false | - |
```

Comment:
Is this completely gone or simply deprecated?

Reply:
I have changed my mind since the code here. Now it will fall back to envvars with a warning. But there *are* usecases where envvars are totally appropriate (and should be documented). I will fix this.